rendermodules/rough_align/apply_rough_alignment_to_montages.py

```diff
@@ -0,0 +1,244 @@
+
+import os
+import json
+import renderapi
+import glob
+import numpy as np
+from ..module.render_module import RenderModule
+from rendermodules.rough_align.schemas import ApplyRoughAlignmentTransformParameters
+from rendermodules.stack.consolidate_transforms import *
```

Comment:
for the sake of readability, it could be helpful be explicit here -- it's not obvious, for instance, that RenderModuleException is being imported from rendermodules.stack.consolidate_transforms